File: 백준/Bronze/2720. 세탁소 사장 동혁/세탁소 사장 동혁.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int t =  sc.nextInt();

        for (int i = 0; i < t; i++) {
            int c =  sc.nextInt();

            int quarter = c/25;
            c = c % 25;
            int dime = c / 10;
            c = c % 10;
            int nickel = c / 5;
            c = c % 5;
            int penny = c;

            System.out.printf("%d %d %d %d\n",quarter,dime,nickel,penny);
        }
    }
}

File: 백준/Bronze/2720. 세탁소 사장 동혁/세탁소 사장 동혁.py
t = int(input())

for i in range(t):
    c = int(input())

    quarter, dime, nickel, penny = 0, 0, 0, 0

    while True:
        quarter = c // 25
        dime = (c % 25) // 10
        nickel = ((c % 25) % 10) // 5
        penny = ((c % 25) % 10) % 5
        break

    print(quarter, dime, nickel, penny)